File: src/main/java/org/cimsbioko/syncadpt/SyncCancelReceiver.java
package org.cimsbioko.syncadpt;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;
import org.cimsbioko.utilities.SyncUtils;

public class SyncCancelReceiver extends BroadcastReceiver {
    @Override
    public void onReceive(Context context, Intent intent) {
        SyncUtils.cancelUpdate(context);
    }
}

File: src/main/kotlin/org/cimsbioko/syncadpt/SyncCancelReceiver.kt
package org.cimsbioko.syncadpt

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent
import org.cimsbioko.utilities.SyncUtils.cancelUpdate

class SyncCancelReceiver : BroadcastReceiver() {
    override fun onReceive(context: Context, intent: Intent) {
        cancelUpdate(context)
    }
}